CardScan/Classes/CreditCardOcr/SSDCreditCardOcr.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SSDCreditCardOcr.swift
+//  CardScan
+//
+//  Created by xaen on 5/15/20.
+//
+import UIKit
+
+@available(iOS 11.2, *)
+class SSDCreditCardOcr: CreditCardOcrImplementation {
+    override func recognizeCard(in fullImage: CGImage, roiRectangle: CGRect) -> CreditCardOcrPrediction {
+        guard let image = croppedImageForSsd(fullCardImage: fullImage, roiRectangle: roiRectangle)
+            else {
+                return CreditCardOcrPrediction.emptyPrediction(cgImage: fullImage)
+        }
+                
+        let ocr = OcrDD()
+        let startTime = Date()
+        let number = ocr.perform(croppedCardImage: image)
+        let duration = -startTime.timeIntervalSinceNow
+        print("DEBUG: Inference time for DD Ocr: ", duration)
```

Comment:
let's remove the debugging print statements or have a flag that defaults to "off" for them